src/HashedSimplify.hs

```diff
@@ -436,30 +401,12 @@ evaluateIfPossibleRules exp@(mp, n) =
 standardize :: Transformation
 standardize = toRecursiveSimplification (noChange . snd)
 
-
-
-{--
---    ==============================================
---    ==  Rotate operation simplification rules   ==
---    ==============================================
----}
 -- | Combining the Rotate
 -- rotate [2] (rotate [1] arg) -> rotate [3] arg
 combineRotateRules :: Modification
 combineRotateRules exp@(mp, n)
   | Rotate outerVal outerN <- retrieveNode n mp
   , Rotate innerVal innerN <- retrieveNode outerN mp =
-        applyDiff mp (unary (Rotate (listAdd outerVal innerVal))) [noChange innerN]
+    applyDiff mp (unary (Rotate (zipWith (+) outerVal innerVal))) [noChange innerN]
```

Comment:
`outerVal` -> `outerAmount`
`innerVal` -> `innerAmount`

Is this more readable?
```haskell
let totalAmount = zipWith (+) outerVal innerVal 
in applyDiff mp (unary (Rotate totalAmount)) [noChange innerN]
```